docs/release.md

```diff
@@ -12,7 +12,7 @@
 ## Directions
 
 ### Initiate the Release
-1. Increment the version number and update the Public API files; both of these steps can be accomplished by running the `tools\IncrementVersion.cmd` script. Then create a pull request with these changes and merge it.
+1. Increment the version number and update the Public API files; both of these steps can be accomplished by running the `tools\IncrementVersion.cmd` script. Then create a pull request with these changes using the `tools\Release\CreatePullRequest.cmd` script and merge the pull request.
```

Comment:
The name is "CreatePullRequest", but the content looks like to create a branch and upload to upstream. Right?
Shall we rename it?